code/modules/fish/fish_tank.dm

```diff
@@ -340,19 +355,43 @@
 		fish_list.Remove(pick(fish_list))			//Kill a random fish
 	update_icon()
 
+/obj/machinery/fishtank/proc/seadevil_eat()
```

Comment:
Might be better in the Sea Devil's `special_interaction()`, if you want to go that way.